docs/turn-howto.md

```diff
@@ -42,65 +38,112 @@ This will install and start a systemd service called `coturn`.
 
         ./configure
 
-    You may need to install `libevent2`: if so, you should do so in
-    the way recommended by your operating system. You can ignore
-    warnings about lack of database support: a database is unnecessary
-    for this purpose.
+    > You may need to install `libevent2`: if so, you should do so in
+    > the way recommended by your operating system. You can ignore
+    > warnings about lack of database support: a database is unnecessary
+    > for this purpose.
 
 1.  Build and install it:
 
         make
         make install
 
+#### Docker installation
+There are coturn docker images available. You can prepare by running
+
+        docker pull instrumentisto/coturn
+
+The first part of the command needed to start your container detached is as follows:
+
+        docker run -d -p 3478:3478/udp -p 3478:3478/tcp \
+        -p <range_start-range_end:range_start-range_end>/udp \
```

Comment:
I notice this is not recommended in https://github.com/instrumentisto/coturn-docker-image#why-so-many-ports-opened. Should we at least link to that advice?